chiselled-jre/Dockerfile.22.04

```diff
@@ -33,7 +33,8 @@ RUN mkdir -p /rootfs \
     && chisel cut --release /opt/chisel-releases --root /rootfs \
         openjdk-8-jre-headless_bins \
         media-types_data \
-        libglib2.0-0_libs \
+	libffi8_libs \
```

Comment:
I have regenerated minimal fileset after dropping glib and this is the result: https://git.launchpad.net/~vpa1977/+git/package-tracker/tree/files/x86_64/minimal?h=jre-8

```
/lib/x86_64-linux-gnu/ld-linux-x86-64.so.2
/lib/x86_64-linux-gnu/libc.so.6
/lib/x86_64-linux-gnu/libgcc_s.so.1
/lib/x86_64-linux-gnu/libm.so.6
/lib/x86_64-linux-gnu/libstdc++.so.6.0.30
/lib/x86_64-linux-gnu/libthread_db.so.1
/lib/x86_64-linux-gnu/libz.so.1.2.13
```
We do not need neither ffi8 nor mount1.

Reply:
alright, removed them